### PR TITLE
BXMSDOC-4893 Managing and monitoring business processes for PAM 7.6

### PR DESCRIPTION
**Pull request by emmurphy1**
Details in epic:
[BXMSDOC-4893](https://issues.redhat.com/browse/BXMSDOC-4893)

### CODE REVIEW

File: doc-content/enterprise-only/processes/process-instance-filtering-proc.adoc [hunk 1]
```diff
@@ -3,60 +3,36 @@
...
-image::processes/DateRangeSearch.png[Search by Date Range]
+* *Name*: Filter process instances based on process definition name.
+* *Definition ID*: The ID of the instance definition.
+* *Deployment ID*: The ID of the instance deployment.
+* *SLA Compliance*: SLA compliance status (*Aborted*, *Met*, *N/A*, *Pending*, and *Violated*).
+* *Parent Process ID*: The ID of the parent process.
+* *Start Date*: Filter process instances based on their creation date.
+* *Last update*: Filter process instances based on their last modified date.
+
+You can also use the *Advanced Filters* option to create custom filters in {CENTRAL}.
```
Comment (kaldesai) [inline doc-content/enterprise-only/processes/process-instance-filtering-proc.adoc:38]: Hey @emmurphy1, I guess, the rendered output link is not reflecting the changes. Can you please attach the updated links?

Comment (emmurphy1) [reply doc-content/enterprise-only/processes/process-instance-filtering-proc.adoc:38]: The links are correct. Check section 4.1.

Comment (kaldesai) [reply doc-content/enterprise-only/processes/process-instance-filtering-proc.adoc:38]: @emmurphy1, Apologies for the same. I was checking different rendered output link.

**Review by kaldesai — COMMENTED** [review]
Everything looks good to me.

**Pull request merged**